suites/upgrade/hammer-x/stress-split-erasure-code-x86_64/distros/ubuntu_14.04.yaml

```diff
@@ -1,3 +0,0 @@
-os_type: ubuntu
-os_version: "14.04"
-arch: x86_64
```

Comment:
You need an arch directory like in https://github.com/ceph/ceph-qa-suite/pull/571/files